packages/fxa-auth-server/lib/routes/recovery-phone.ts

```diff
@@ -58,9 +62,32 @@ class RecoveryPhoneHandler {
 
     await this.customs.check(request, email, 'recoveryPhoneSendCode');
 
-    let status = false;
+    const ftlIdMsgs = [
+      {
+        id: 'recovery-phone-signin-sms-body-part-1',
+        message: 'Mozilla recovery code:',
+      },
+      {
+        id: 'recovery-phone-sms-body-part-2',
+        message: 'Expires in 5 minutes. Do not share this code.',
+      },
+    ];
+    const localizedStrings = await this.localizer.localizeStrings(
+      request.app.locale,
+      ftlIdMsgs
+    );
+
+    const localizedMessageBody = {
```

Comment:
Not sure, but I thought there was a way to have place holders in l10n templates? The issue with this is that the translation might need to change the word order. For example German and English translations will often have the words in different places for a given sentence.

Reply:
That's true, and I'd modified the formatted message a bit to work around that variation, but your suggestion of using a template function is cleaner and will give localizers more flexibility.